app/packages/state/src/recoil/distributions.ts

```diff
@@ -35,11 +35,26 @@ import {
   withPath,
 } from "@fiftyone/utilities";
 import { State } from "./types";
+import { extendedSelection } from "./atoms";
+import { filters } from "./filters";
+import { groupSlice, groupStatistics } from "./groups";
 
 export type AggregationResponseFrom<
   TAggregate extends { response: { aggregate: readonly unknown[] } }
 > = TAggregate["response"]["aggregate"][0];
 
+const extendedViewForm = selector({
+  key: "extendedViewForm",
+  get: ({ get }) => {
+    return {
+      sampleIds: get(extendedSelection),
```

Comment:
I'm making a selector for creating saved views that looks similar to this one, should we maybe make a form selector in `Selectors.ts` to reuse across the app? That way we don't have to check individual components when something is changed.
```
        send((session) =>
            saveView({
              onError,
              variables: {
                viewName: name,
                datasetName: datasetNameValue,
                viewStages: view,
                form: {
                  filters: snapshot.getLoadable(filters).contents,
                  sampleIds: [
                    ...snapshot.getLoadable(selectedSamples).contents,
                  ],
                  labels: snapshot.getLoadable(selectedLabelList).contents,
                  extended: snapshot.getLoadable(fos.extendedStages).contents,
                  slice:
                    snapshot.getLoadable(resolvedGroupSlice(false)).contents ||
                    null,
                },
                description,
                color,
                subscription,
                session,
              },

Reply:
Yeah, if this selector overlaps with the selector you're going to create then we can probably use a common selector for both.

Reply:
Currently, the aggregation form does not have `extended` but maybe we can a derived selector which is subset of above for form

Reply:
Yes, I would agree with a move to `selectors.ts` if you want to make that change @kaixi-wang